### PR TITLE
118 refactor loginterpreter

### PR DESCRIPTION
**Pull request by samer-12**

### CODE REVIEW

**Review by hellesvaasand — APPROVED** [review]
🌟

File: extractor/logInterpreter.py [hunk 1]
```diff
@@ -2,6 +2,11 @@
 from collections import defaultdict
 
 class LogInterpreter:
+
+    
+    def __init__(self):
+        self.relatedSessionFormat = "X-Gt-MBN-SessionId"
```
Comment (moltubaer) [inline extractor/logInterpreter.py:8]: Change to a more flexible regex

**Review by moltubaer — CHANGES_REQUESTED** [review]
see regex comment

**Review by moltubaer — APPROVED** [review]
👍👍👍

**Pull request merged**